core/user.php

```diff
@@ -196,7 +196,7 @@ public function check_password($password) {
 
 	public function load_from_email($email) {
 		//echo "<h5>Loading user object from db with email {$email}</h5>";
-		$query = "select * from users where email=?";
+		$query = "select * from users where email=? and state>=1";
```

Comment:
revert this change - state will be added to this function as well as load_from_id to be used wherever they are called from